include/game.h

```diff
@@ -988,6 +988,26 @@ typedef struct {
     /* 0x0E */ u16 unk0E;
 } RelicOrb; /* size=0x10 */
 
+typedef struct {
+    /* 0x00 */ const char* D_801C24CC;   // pointer to dialogue next character
```

Comment:
I know this was pre-existing but seems like if we know what these do we ought to just name them?

Reply:
I rather leave them because @Xeeynamo was working on these functions and maybe he needs to conserve the addresses for something, and i don't want to disrupt the process he's going through to understand them.

Reply:
Alright! just renamed them, let me know if everything looks good.